tests/Queue/Failed/DatabaseFailedJobProviderTest.php

```diff
@@ -77,7 +77,7 @@ public function testAll(): void
         $all = $this->getProvider()->all();
 
         $this->assertCount(5, $all);
-        $this->assertEquals(new ObjectId(sprintf('%024d', 5)), $all[0]->_id);
+        $this->assertEquals(new ObjectId(sprintf('%024d', 5)), $all[0]->id);
         $this->assertEquals(sprintf('%024d', 5), $all[0]->id, 'id field is added for compatibility with DatabaseFailedJobProvider');
```

Comment:
Unrelated to the diff, but would this benefit from using `assertSame` to also check that a string type is used?

Reply:
They are not the same. 2 distinct objects line 80.
I add assertions on the type of the "id" property.